observability/logging/api_reference/logging-5-7-reference.adoc

```diff
@@ -1,7 +1,1310 @@
+|requests|object|  *(optional)* Requests describes the minimum amount of compute resources required.
+|======================
+
+=== .spec.visualization.kibana.resources.limits
+
+Type:: object
+
+=== .spec.visualization.kibana.resources.requests
+
+Type:: object
+
+=== .spec.visualization.kibana.tolerations[]
+
+[IMPORTANT]
+====
+This API key has been deprecated and is planned for removal in a future release. For more information, see the release notes for logging on Red{nbsp}Hat OpenShift.
+====
+
+Type:: array
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|effect|string|  *(optional)* Effect indicates the taint effect to match. Empty means match all taint effects.
+|key|string|  *(optional)* Key is the taint key that the toleration applies to. Empty means match all taint keys.
+|operator|string|  *(optional)* Operator represents a key&#39;s relationship to the value.
+|tolerationSeconds|int|  *(optional)* TolerationSeconds represents the period of time the toleration (which must be
+|value|string|  *(optional)* Value is the taint value the toleration matches to.
+|======================
+
+=== .spec.visualization.kibana.tolerations[].tolerationSeconds
+
+Type:: int
+
+=== .spec.visualization.nodeSelector
+
+Type:: object
+
+=== .spec.visualization.ocpConsole
+
+Type:: object
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|logsLimit|int|  *(optional)* LogsLimit is the max number of entries returned for a query.
+|timeout|string|  *(optional)* Timeout is the max duration before a query timeout
+|======================
+
+=== .spec.visualization.tolerations[]
+
+Type:: array
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|effect|string|  *(optional)* Effect indicates the taint effect to match. Empty means match all taint effects.
+|key|string|  *(optional)* Key is the taint key that the toleration applies to. Empty means match all taint keys.
+|operator|string|  *(optional)* Operator represents a key&#39;s relationship to the value.
+|tolerationSeconds|int|  *(optional)* TolerationSeconds represents the period of time the toleration (which must be
+|value|string|  *(optional)* Value is the taint value the toleration matches to.
+|======================
+
+=== .spec.visualization.tolerations[].tolerationSeconds
+
+Type:: int
+
+=== .status
+
+ClusterLoggingStatus defines the observed state of ClusterLogging
+
+Type:: object
+
+[options="header"]
+|======================
+|Property|Type|Description
+
+|collection|object| **(DEPRECATED)** *(optional)* Deprecated.
+|conditions|object|  *(optional)*
+|curation|object|  *(optional)*
+|logStore|object|  *(optional)*
+|visualization|object|  *(optional)*
+|======================
+
+=== .status.collection
+
+[IMPORTANT]
+====
+This API key has been deprecated and is planned for removal in a future release. For more information, see the release notes for logging on Red{nbsp}Hat OpenShift.
```

Comment:
I called this out previously - I leave it to you to determine if the term "{OCP}" even CAN be substituted - could be a problem with the curly braces?